libqtile/backend/x11/core.py

```diff
@@ -736,6 +737,9 @@ def handle_MappingNotify(self, event):  # noqa: N802
     def handle_MapRequest(self, event) -> None:  # noqa: N802
         assert self.qtile is not None
 
+        if event.window in self.override_redirect_map:
```

Comment:
If we're mapping it then the window will be in windows_map so we should remove it here.